pyspark/bigdl/util/common.py

```diff
@@ -434,6 +434,18 @@ def _test():
     if failure_count:
         exit(-1)
 
+class TransformationError(BaseException):
+    pass
+
+def ndarray_to_rdd(X, y):
```

Comment:
add another parameter: `numSlices=None` to control the partition num?

Reply:
I've fixed two issues you've mentioned, but got a question about this one:
Is this partition number a pre-build function of Spark Contex, of you want to to simple add it up manually? (like slice ndarrays to partitions and parallelize each of them)